node/src/metrics.rs

```diff
@@ -61,6 +61,106 @@ pub(crate) static TRIPLE_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub(crate) static PRESIGNATURE_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "multichain_presignature_latency_sec",
+        "Latency of multichain presignature generation, start from starting generation, end when presignature generation complete.",
+        &["node_account_id"],
+        Some(exponential_buckets(1.0, 1.5, 20).unwrap()),
```

Comment:
Why do we need this?

Reply:
because the latencies are grouped into buckets, in this case it's buckets exponentials, [1,1.5), [1.5, 2.25) and so on, the last bucket will cover +inf. We could also do linear buckets if we want, but that will not cover +inf.